Automattic/wp-calypso
# Fix/annual plan view missing per monthly

#### Changes proposed in this Pull Request

* The per month text which is missing in the mobile view was added back


| Before            | After               |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/3422709/106879324-02879400-6701-11eb-8c2e-bce0dc3b7102.png" data-canonical-src="https://user-images.githubusercontent.com/3422709/106879324-02879400-6701-11eb-8c2e-bce0dc3b7102.png" width="300"  /> | <img width="384" alt="Screenshot 2021-02-05 at 4 07 44 PM" src="https://user-images.githubusercontent.com/1269602/107023246-5ca15b80-67cc-11eb-8926-a6ac8d65aa9a.png"> |
|| <img width="399" alt="Screenshot 2021-02-05 at 4 07 52 PM" src="https://user-images.githubusercontent.com/1269602/107023309-73e04900-67cc-11eb-8763-96df96d22adf.png"> |



#### Testing instructions

- Activate experiment locally from link available at pcbrnV-XN-p2
- Go to the plans page and make sure that the price per monthly next to the number in the mobile view
- Make sure the desktop view hasn't changed
- make sure /plans page hasn't changed

Related to #48893 #48728
